### ISSUE TITLE
Fatal signal 11 (SIGSEGV) on Android 4.3 Samsung S3

### ISSUE THREAD

**Issue by oriharel** [jxcore/jxcore#580]
It's actually a re-open of https://github.com/jxcore/jxcore/issues/494
This is with JXCore 0.3.0.6. Here are the logs:

https://drive.google.com/file/d/0B0bdCx3BDBCLSXpXdEJvamtDODg/view?usp=sharing

**Comment by obastemur** [comment]
Hmm can you try standalone 0.3.0.6 binary from adb terminal?

**Comment by oriharel** [comment]
Can you explain? what exactly you want me to do? It doesn't reproduces every time btw.

**Comment by obastemur** [comment]
Download Android ARM standalone from jxcore.com/downloads

```
adb push jx /data/local/tmp/
adb shell
cd /data/local/tmp/
chmod 777 jx
./jx -e "0"
```

**Comment by obastemur** [comment]
>  It doesn't reproduces every time btw.

Device has the stock firmware ?

**Comment by oriharel** [comment]
Thank Oz. I'll try it when I'll get to that phone again. Yes, It's running Stock Samsung build (it's not even rooted).

**Comment by oriharel** [comment]
The command processed without an output (which probably means no sigsegv, right?)

**Comment by obastemur** [comment]
Yes. You might run it several times to see if it's happening. We've just released 0.3.0.7, you might give a shot. At this moment have no idea what happening since there is nothing special on log you've sent.

**Comment by ktrzeciaknubisa** [comment]
Just for clarity: shouldn't we have this conversation at jxcore-cordova repo? :)

@oriharel which jxcore-cordova binaries are you using? 

I myself have a Samsung S3 (Android 4.1.2, stock, non-rooted) and I do have occasionally problems too with running cordova samples. 

```bash
A/libc(19260): @@@ ABORTING: INVALID HEAP ADDRESS IN dlfree addr=0x60efe4f8
A/libc(19260): Fatal signal 11 (SIGSEGV) at 0xdeadbaad (code=1), thread 19260 (m.example.hello)
```

I'm taking binaries from [jxcore-cordova-release 0.0.5](https://github.com/jxcore/jxcore-cordova-release/tree/master/0.0.5).

It does not happen each time, but what i noticed: when it happens it is always deployed from command-line `cordova run android`. I never saw the same problem when running the app through Eclipse (so far).

Standalone jx 0.3.0.7 works alright for me.

**Comment by obastemur** [comment]
@oriharel any update on this one ?

**Comment by oriharel** [comment]
No. We don't have too much coverage at our company but right now we only see it in this S3/Android 4.3 devices. Not sure if it's a Samsung issue or Android 4.3 issue. @ktrzeciaknubisa I'm not using jxcore-cordova, using the Android FAT binaries from the download page.

**Comment by obastemur** [comment]
@ktrzeciaknubisa can you repeat it with `0.0.8` ? to see if it happens with `cordova run android` ?

**Comment by ktrzeciaknubisa** [comment]
Well, it works only 3 times per 5 runs (twice it didn't work with same error as above).

**Comment by obastemur** [comment]
@ktrzeciaknubisa are you able to reproduce this issue with `jx standalone` binary ?

**Comment by ktrzeciaknubisa** [comment]
Nope. Before I was trying with 0.3.0.7 and it was ok. Now I've tried with 0.3.1.0 and it still works (all tests from test/jxcore). And it is not even a rooted phone.
